가운데글자가져오기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    const ss = s.split('');
+    const sss = s.length;
+    const ssss = sss-(sss/2);
+    if (sss%2 !== 0){
+        return s.charAt(ssss);
+    }
+    else{
+        return s.substring((sss-ssss)-1, sss-ssss+1);
+     }
```

Comment:
```suggestion
    if (sss % 2 !== 0) return s.charAt(ssss);
    else return s.substring((sss-ssss)-1, sss-ssss+1);  
```
if 문의 실행문이 한줄일땐 중괄호 생략이 가능합니다.
공백 컨벤션 지켜주세요